sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue.cs

```diff
@@ -218,6 +218,23 @@ public long AsInt64()
             return _fieldValue.ValueInteger.Value;
         }
 
+        /// <summary>
+        /// Gets the value of the field as a <see cref="long"/>.
+        /// If the operation fails, null value will be returned.
+        /// </summary>
+        /// <returns>The value of the field converted to a <see cref="long"/>, or null in case of failure.</returns>
+        public long? AsInt64OrNull()
+        {
+            try
+            {
+                return AsInt64();
+            }
+            catch
```

Comment:
Maybe we should be more specific about the type of exception we're expecting here (I think only `InvalidOperationException` is expected to happen, right?). Otherwise, if we have an unexpected exception (`NullReferenceException` or `OutOfMemoryException`, for example), we could be potentially hiding a client bug or an error in user code.